cogs/tld.py

```diff
@@ -0,0 +1,63 @@
+import random
+
+import voxelbotutils as utils
+
+
+class TldCommands(utils.Cog):
```

Comment:
This wants to be `TLDCommands` - do your caps like that for capitals in PascalCase.